README.md

```diff
@@ -71,6 +71,8 @@ assert(record1.a === 1);
 assert(record1["a"] === 1);
 assert(record1 !== record2);
 assert(record2 === #{ a: 1, c: 3, b: 5 });
+assert(record1?.a === 1);
+assert(record1?.d ?? 5 === 5);
 ```
```

Comment:
might be too to show an example where the optional chain is called on a nullish LHS

Reply:
yes, makes sense thanks!

Reply:
Note that this still isn’t here; i mean something that’s the equivalent of `null?.something` or `undefined?.something` - without that the “optional” part is useless.

Reply:
oh sorry I did not get it like this, so something like `record1.d?.a === undefined`? Otherwise I could use your help to write that example because unfortunately I feel like I'm missing something obvious.

Reply:
Yes, exactly - basically, something nullish in the record, that someone tries to optionally chain off of.